docs/source/docs/hardware/selecting-hardware.md

```diff
@@ -1,6 +1,6 @@
 # Selecting Hardware
 
-In order to use PhotonVision, you need a coprocessor and a camera. This page will help you select the right hardware for your team depending on your budget, needs, and experience.
```

Comment:
does this not make things more confusing for users? you should tell them where they can actually find product reccomendations if you remove them all from the selecting hardware page